src/customer-service/Factories/CustomerFactory.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using Microsoft.Data.Sqlite;
+using customer_service.Data;
+using customer_service.Models;
+
+namespace customer_service.Factories

```

Comment:
@MattKraatz and I have been naming our factories namespace `customer_service` instead of the Visual Studio default `customer_service.Factories`